### PR TITLE
WMCO: 240.0.0.0 should not be used for clusters that will enable Windows workload

### PR DESCRIPTION
**Pull request by mburke5678**
Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OCPBUGS-34614

Link to docs preview:
WMCO release notes ->[Known limitations](https://77452--ocpdocs-pr.netlify.app/openshift-enterprise/latest/windows_containers/windows-containers-release-notes-6-x#windows-containers-release-notes-limitations_windows-containers-release-notes) -- 8th bullet is new
Enabling Windows containers -> [Installing the Windows Machine Config Operator](https://77452--ocpdocs-pr.netlify.app/openshift-enterprise/latest/windows_containers/enabling-windows-container-workloads#installing-the-wmco) -- New note.

QE review:
- [ ] QE has approved this change.

Additional information:
https://issues.redhat.com/browse/OCPBUGS-32256

### CODE REVIEW

**Comment by mburke5678** [comment]
@sebsoto @rrasouli PTAL

File: modules/windows-containers-release-notes-limitations.adoc [hunk 1]
```diff
@@ -31,6 +31,8 @@ Note the following limitations when working with Windows nodes managed by the WM
 
 * {productwinc} does not support any Windows operating system language other than English (United States). 
 
+* Due to a limitation within the Windows operating system, Class E `clusterNetwork` CIDR (240.0.0.0) addresses are not compatible with Windows nodes.
```
Comment (agantony) [inline modules/windows-containers-release-notes-limitations.adoc:34]:
```suggestion
* Due to a limitation within the Windows operating system, `clusterNetwork` CIDR addresses of class E such as `240.0.0.0` are not compatible with Windows nodes.
```

File: windows_containers/enabling-windows-container-workloads.adoc [hunk 2]
```diff
@@ -40,6 +40,11 @@ Windows instances deployed by the WMCO are configured with the containerd contai
 
 You can install the Windows Machine Config Operator using either the web console or OpenShift CLI (`oc`).
 
+[NOTE]
+====
+Due to a limitation within the Windows operating system, Class E `clusterNetwork` CIDR (240.0.0.0) addresses are not compatible with Windows nodes.
```
Comment (agantony) [inline windows_containers/enabling-windows-container-workloads.adoc:45]:
```suggestion
Due to a limitation within the Windows operating system, `clusterNetwork` CIDR addresses of class E such as `240.0.0.0` are not compatible with Windows nodes.
```

**Review by agantony — COMMENTED** [review]
Added a few suggestions for your consideration; overall lgtm!

**Pull request merged**